app/src/main/res/layout/fragment_crash_reporter.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/photonGrey10">
+
+    <View
```

Comment:
If this image isn't important for accessibility, we should mark it not important for accessibility. Also, it feels odd to use a View with a background instead of an ImageView with a src attribute. One gets additional fields with an ImageView, like scaleType, adjustViewBounds, and other transformations. Background always scales to the size of the View.